langlib/lang.int/src/main/ballerina/int.bal

```diff
@@ -90,6 +95,14 @@ public isolated function abs(int n) returns int = @java:Method {
 
 # Returns sum of zero or more int values.
 #
+# ```ballerina
+# // Retrieving the total out of the list of integers
+# int totalValue = int:sum(10, 20, 30, 40);
+#
+# int initialScore = 4;
+# // Retrieving the total by adding the list of integers with the initial value passed
+# int totalScore = initialScore.sum(5, 6, 7);
```

Comment:
`float`,`decimal` langlib examples PR  purposed three examples for `sum`, `min`, `max`.
Can you try to add examples for different scenarios.

https://github.com/ballerina-platform/ballerina-lang/pull/38920
https://github.com/ballerina-platform/ballerina-lang/pull/38905